server/endpoint/deferrer/endpoint.go

```diff
@@ -0,0 +1,86 @@
+package deferrer
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	kitendpoint "github.com/go-kit/kit/endpoint"
+	kithttp "github.com/go-kit/kit/transport/http"
+)
+
+const (
+	// Method is the HTTP method this endpoint is registered for.
+	Method = "GET"
+	// Name identifies the endpoint. It is aligned to the package path.
+	Name = "deferrer"
+	// Path is the HTTP request path this endpoint is registered for.
+	Path = "/v1/defer/"
+)
+
+// Config represents the configuration used to create a lister endpoint.
+type Config struct {
+	// Dependencies.
+	Logger micrologger.Logger
+}
+
+type Endpoint struct {
+	logger micrologger.Logger
+}
+
+// New creates a new configured lister object.
+func New(config Config) (*Endpoint, error) {
+	// Dependencies.
+	if config.Logger == nil {
+		return nil, microerror.Maskf(invalidConfigError, "%T.Logger must not be empty", config)
+	}
+
+	e := &Endpoint{
+		logger: config.Logger,
+	}
+
+	return e, nil
+}
+
+func (e *Endpoint) Decoder() kithttp.DecodeRequestFunc {
+	return func(ctx context.Context, r *http.Request) (interface{}, error) {
+		return nil, nil
+	}
+}
+
+func (e *Endpoint) Encoder() kithttp.EncodeResponseFunc {
+	return func(ctx context.Context, w http.ResponseWriter, response interface{}) error {
+		w.Header().Set("Content-Type", "text/plain; charset=utf-8")
+		w.WriteHeader(http.StatusOK)
+		b, ok := response.([]byte)
+		if !ok {
+			return microerror.Mask(invalidResponseTypeError)
+		}
+		_, err := w.Write(b)
+		return err
+	}
+}
+
+func (e *Endpoint) Endpoint() kitendpoint.Endpoint {
+	return func(ctx context.Context, request interface{}) (interface{}, error) {
+		return []byte("no"), nil
```

Comment:
When we start to integrate that the default can be `yes` to just keep the termination as it is even if it is build into the process.

Reply:
I think by this you mean that this service shouldn't block current `qemu-shutdown` behaviour when we start integrating this? This is what I noted in description that this is a bit non-intuitive, but reflects the semantic question by client.

Do you have ideas for better wording (for both resource and response) so that it would be more intuitive?

Reply:
I see. The question is if the deferrer says "yes" shutdown, or "yes" defer or something like this. `GET /v1/defer -> "no"` is pretty obvious. I was just not paying enough attention here. Lets go with how it is. All good.